packages/number-line/controller/src/index.js

```diff
@@ -46,8 +46,18 @@ export function outcome(model, session, env) {
         accumulateAnswer(model.correctResponse),
         0
       );
+
+      let numIncorrect = 0;
+      if((session.answer || []).length > model.correctResponse.length) {
+         numIncorrect = (session.answer || []).length - model.correctResponse.length;
+      }
+
       const total = model.correctResponse.length;
```

Comment:
move total above (line 49) and use that constant instead of model.correctResponse.length

Reply:
also, could total be 0?